tests/k8s/Vagrantfile

```diff
@@ -0,0 +1,58 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.require_version ">= 1.8.3"
+
+$build = <<SCRIPT
+docker run -d -p 5000:5000 registry
+cd /home/vagrant/go/src/github.com/cilium/cilium/
+docker build -t cilium:build_test /home/vagrant/go/src/github.com/cilium/cilium
+export ID=$(docker images | grep "build_test" | awk '{print $3}')
+docker tag $ID localhost:5000/cilium:build_test
+docker push localhost:5000/cilium:build_test
+SCRIPT
+
+$k8s = <<SCRIPT
+sudo apt-get update && sudo apt-get install -y apt-transport-https
+sudo touch /etc/apt/sources.list.d/kubernetes.list
+curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg  | sudo apt-key add -
+sudo bash -c "cat <<EOF > /etc/apt/sources.list.d/kubernetes.list
+deb http://apt.kubernetes.io/ kubernetes-xenial main
+EOF
+"
+sudo apt-get update
+sudo apt-get install -y docker-engine
```

Comment:
Is it possible to specify the version of docker-engine, kubelet, kubeadm, kubectl and kubernetes-cni?